lib/api.go

```diff
@@ -30,37 +34,37 @@ func streaming(o *observable.Observable, w http.ResponseWriter, event string, ca
 		return
 	}
 
-	closeChan := make(chan bool)
+	consumerChan := make(chan struct{})
```

Comment:
There was a problem when several goroutine runs concurrently.
If a event occur during `o.Off(event, observerFunc)` is being called,
the callback function trying to send message to `messageChan`.
However messageChan receiver already disappear.

Because of this reason, the consumerChan is introduced to notify the messageChan receiver is gone.